### PR TITLE
- Add PSS exceptions.

### PR DESCRIPTION
**Pull request by whites11**
This PR adds support for legacy release 19.3.0 with PSS

towards: https://github.com/giantswarm/giantswarm/issues/28582

### CODE REVIEW

**Review by T-Kukawka — APPROVED** [review]

File: helm/aws-load-balancer-controller/templates/pss-exceptions.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{{- if .Values.global.podSecurityStandards.enforced }}
+apiVersion: kyverno.io/v2alpha1
+kind: PolicyException
+metadata:
+  name: {{ include "aws-load-balancer-controller.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "aws-load-balancer-controller.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": "pre-install,pre-upgrade"
```
Comment (fiunchinho) [inline helm/aws-load-balancer-controller/templates/pss-exceptions.yaml:10]: why is this necessary?

File: helm/aws-load-balancer-controller/templates/pss-exceptions.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+spec:
+  exceptions:
+    - policyName: disallow-capabilities-strict
+      ruleNames:
+        - require-drop-all
```
Comment (mcharriere) [inline helm/aws-load-balancer-controller/templates/pss-exceptions.yaml:15]: why do you need an exception for this? I don't think this app requires any capability
